docs/add-non-um-users.html

```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
-title: Add Non-UM Users
+title: Add Non&#8209;UM Users
 ---
 
 <div id="content-wrap">
-  <h1>Add Non-UM Users Documentation</h1>
+  <h1>Add Non&#8209;UM Users Documentation</h1>
```

Comment:
Maybe I'm missing something, but the non-breaking special character doesn't seem to be rendered properly in the documentation (the navigation links look okay though).
![Screen Shot 2021-12-13 at 12 33 17 PM](https://user-images.githubusercontent.com/35741256/145860784-634f9c1f-d8d8-4617-870e-f7bf5f2ba017.png)
.

Reply:
Hmm... I am not seeing this with browsers(chrome, safari, edge, FF) i tested. I really don't want to complicate this. I chose to go back to regular (`-`)
![Screen Shot 2021-12-14 at 8 45 25 AM](https://user-images.githubusercontent.com/8579775/146010944-ca8391b1-d358-4538-b9f9-d31b96cb6423.png)

Reply:
@ssciolla I made some [change](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/268/commits/a18d450d286755ec33fc60918e7a0c0782a80eeb). See if this is any better. I agree not all needs to be encoded.